user_data/strategies/AdvancedHighPerformanceStrategy.py

```
@@ -0,0 +1,164 @@
+# --- Do not remove these libs ---
+from freqtrade.strategy import IStrategy, merge_informative_pair
```

Ruff (0.11.9)

`freqtrade.strategy.merge_informative_pair` imported but unused

Remove unused import: `freqtrade.strategy.merge_informative_pair`

(F401)